http/vertx-web-validation/pom.xml

```
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.ts.qe</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <artifactId>vertx-web-validation</artifactId>
+    <packaging>jar</packaging>
+    <name>Quarkus QE TS: HTTP: vertx-web-validation</name>
```

```suggestion
    <name>Quarkus QE TS: HTTP: Vert.x Web Validation</name>
```
based on https://vertx.io/docs/vertx-web-validation/java/